.pylintrc

```diff
@@ -25,10 +25,6 @@ jobs=0
 # Set reasonable line length
 max-line-length=120
 
-# When enabled, pylint would attempt to guess common misconfiguration and emit
-# user-friendly hints instead of false-positive error messages.
-suggestion-mode=yes
```

Comment:
There is now a warning barking about this existing because the feature is always on now.